package.json

```diff
@@ -5,18 +5,22 @@
     "test": "npm run check-markdown && npm run check-examples",
     "check-markdown": "remark . -f -r .github/remark.yaml -i .remarkignore",
     "format-markdown": "remark . -f -r .github/remark.yaml -i .remarkignore -o",
+    "prettier-format-markdown": "prettier --write '**/*.md'",
```

Comment:
I'm guessing this is the culprit for most markdown changes? Maybe it is missing some options to align with what was already there.

Reply:
the culprit was actually npm run format-markdown using a different version of something from the npm lint. I added prettier to try to see if that fixed warnings before I knew there was a discrepancy. removed prettier and did a hard reset before all markdown changes.